lib/service-manager.js

```diff
@@ -202,23 +202,24 @@ prototype.onApiRequest = function(req, callback) {
 // XXX(sam) this includes updates made from PM, so setStartOptions() may
 // need to filter out calls if there is no change.
 prototype.onInstanceUpdate = function(instance, callback) {
-  // XXX(rmg) there's a bug in here somewhere.. setImmediate breaks things
-  return process.nextTick(callback);
-  /* FIXME set-size is broken ATM
   var self = this;
-  var cpus = 'STRONGLOOP_CLUSTER' in process.env ? // XXX should be service.env?
-    process.env.STRONGLOOP_CLUSTER : 'CPU';
+  var CPUS = 'STRONGLOOP_CLUSTER' in process.env ?
```

Comment:
No, slc ctl set-size works fine, I've banged on it pretty hard.

I actually suspect this to be dead code, but I can't be sure, and its existed since before multiapp.